--- po/Makefile	2008-12-27 04:06:01.000000000 +0800
+++ po/Makefile.new	2012-06-24 23:51:15.393941025 +0900
@@ -1,5 +1,4 @@
 PLUGIN = wavelet-decompose
-LOCALEDIR = /usr/share/locale
 LANGUAGES = de ru it pl
 
 # END CONFIG ##################################################################
@@ -24,7 +23,7 @@
 
 install: $(LANGUAGES)
 	for L in $(LANGUAGES); \
-	do install -v -m 0644 $$L.mo "$(LOCALEDIR)/$$L/LC_MESSAGES/gimp20-$(PLUGIN)-plug-in.mo"; \
+	do install -D -m 0644 $$L.mo "$(LOCALEDIR)/$$L/LC_MESSAGES/gimp20-$(PLUGIN)-plug-in.mo"; \
 done
 
 uninstall: $(LANGUAGES)
